activities/activity_nick_grokhowsky.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+
+# Function to show significantly high and low pixels
+def run_significantValues(elev, env, **kwargs):
+    gs.run_command(
+        "r.neighbors", input=elev, output="smooth", method="average", flags="c", env=env
+    )
+    # Calculate univariate statistics
+    stats = gs.parse_command("r.univar", map="smooth", flags="g", env=env)
+
+    # Calculate mean, standard deviation, and thresholds for high and low
+    mean = float(stats["mean"])
+    sd = float(stats["stddev"])
+    high = str(mean + (1 * sd))
+    low = str(mean - (1 * sd))
+
+    # Using map algebra create a new raster map of highest and lowest pixel values versus all others
+    gs.mapcalc(
+        f"smooth = if(smooth < {low}, 0, if(smooth > {high}, 1, smooth))",
+        overwrite=True,
+    )
+
+    # Change colors for high and low maps
+    gs.run_command("r.colors", map="smooth", color="elevation", env=env)
```

Comment:
Why would you want to combine categories 0 and 1 together with elevation data? Why not
```
 f"smooth = if(smooth < {low}, 0, if(smooth > {high}, 1, null()))"
```

Reply:
You are right.  I keep forgetting this is meant to be displayed on the tangible landscape, and my thought was the elevation data/background would give context.  I pushed a new version with null values.  Thanks for putting up with all of this craziness.

Reply:
> I keep forgetting this is meant to be displayed on the tangible landscape, and my thought was the elevation data/background would give context.

Just as a general note, you may want to be more careful here: raster algebra can be used to support visualization, but what you are saying would be much better done by overlaying these two rasters during visualization (so in this context by two different d.rast commands in the JSON config file, in general, probably by adding two layers in GUI).